build.gradle

```diff
@@ -322,6 +323,17 @@ project('segmentstore:storage:impl') {
 }
 
 project ('bindings') {
+    configurations.all {
+        resolutionStrategy {
+            // We need to force this version as dependency, since ':test:testcommon' brings in a newer version of 
```

Comment:
This comment makes no sense. If there is a conflict with testcommon why is the `bindings` version being changed? This fix should go into testcommon. Because testcommon is not depended on by any non-test target there should be no conflict.

Reply:
The problem really is that the dependencies `HDFS` and `S3` of the `bindings` sub-project bring in an older version of Gson `2.5` (2015) transitively, while the `client` and `testcommon` sub-projects depend upon a newer version `2.8.5` (2018). I don't think it's a good idea to downgrade the Gson dependency used by `client` and `testcommon` subprojects, just because the old versions of `HDFS` and `S3` we use in turn use a rather old version of `Gson`. Also, we already have plans to upgrade HDFS and S3 to newer versions, which should allow us to remove this config. So, IMO this change needs to stay.

What do you say?

Reply:
Now that you have changed the code both :client and :test:testcommon can remove the dependency all together.

Reply:
Yeah, that makes sense. I've removed this configuration section now and the dependency to `testcommon`.